src/Components/Article.jsx

```diff
@@ -0,0 +1,92 @@
+import { useParams } from "react-router-dom"
+import { useState, useEffect } from "react"
+import { Card } from "react-bootstrap"
+import { fetchArticle, fetchCommentsForArticle, patchArticleVotes } from "../api"
+
+function Article(){
+    const { article_id} = useParams()
+
+    const [article, setArticle] = useState("")
+    const [comments, setComments] = useState("")
+    const [articleVotes, setArticleVotes] = useState("")
+    const [haveMadeVote, setHaveMadeVote] = useState(false)
```

Comment:
Although its not part of this ticket, I'd have some general loading state in this component